### ISSUE TITLE
[ter-prefer-arrow-callback] false positive for named function expression

### ISSUE THREAD

**Issue by ikokostya** [buzinas/tslint-eslint-rules#187]
**Versions**

```
├── tslint@4.4.2 
├── tslint-eslint-rules@3.2.3 
└── typescript@2.2.1 
```

**Code example**

```ts
function decorate(fn: Function): Function {
    return fn;
}

const foo = decorate(function bar() {});
```

**Actual behavior**

```
Unexpected function expression. (ter-prefer-arrow-callback)
  3 | }
  4 | 
> 5 | const foo = decorate(function bar() {});
    |                     ^
  6 | 
```

**Expected behavior**

No errors, because `bar` is **named** function expression.

**Comment by ikokostya** [comment]
Oh, my bad. Default value of `allowNamedFunctions` option for this rule is `false` by default https://github.com/buzinas/tslint-eslint-rules/blob/master/src/docs/rules/terPreferArrowCallbackRule.md
